src/data.js

```diff
@@ -21,3 +21,19 @@ export const getComments = (article_id) => {
     return data.comments;
   });
 };
+
```

Comment:
Nice patch requests, though these could be refactored into one function that takes an argument of how many votes to increment by (i.e. -1 or 1) 🤔